plugins/fluentd_telemetry_plugin/src/streamer.py

```diff
@@ -95,6 +96,18 @@ def ufm_telemetry_endpoints(self):
         xdr_ports_types = self.ufm_telemetry_xdr_ports_types.split(splitter)
         intervals = self.streaming_interval.split(splitter)
         msg_tags = self.fluentd_msg_tag.split(splitter)
+
+        error_message = ""
+        expected_amount = len(hosts)
+        for name, array in [("port", ports), ("url", urls), ("interval" ,intervals), ("message_tag_name", msg_tags),\
```

Comment:
I think you should validate only the required fields that are defined in the schema https://github.com/Mellanox/ufm_sdk_3.0/blob/main/plugins/fluentd_telemetry_plugin/src/schemas/set_conf.schema.json#L59

the optional fields such as the message_tag_name or xdr_mode should be included in the validation only if they have data